packages/lib/src/vis/shared/SelectionMesh.tsx

```diff
@@ -11,7 +11,11 @@ import VisMesh from './VisMesh';
 interface Props extends MeshProps {
   onSelection?: (startPoint: Vector2, endPoint: Vector2) => void;
   modifierKey?: ModifierKey;
-  children: (startPoint: Vector2, endPoint: Vector2) => ReactElement;
+  children: (
+    startPoint: Vector2,
+    endPoint: Vector2,
+    isDragging: boolean
+  ) => ReactElement | null;
```

Comment:
As explained above, the render props function was changed to include the `isDragging` boolean that tells if the user is selecting/dragging the selection.

This allows to remove the selection component in `ZoomSelectionMesh` when the selection was finished.